src/app/news/news.scss

```diff
@@ -1,14 +1,12 @@
-$max-width: 640px;
+@import '../variables';
 
 .view-container {
   display: grid;
-  grid-template-rows: auto 1fr;
   grid-template-areas: 'form' 'news';
```

Comment:
For `grid-template-areas` rows are separated by quotes and columns by spaces within quotes.  Here you've defined two rows as the template, but you should change it to `'form news'`